worker.go

```diff
@@ -54,6 +57,7 @@ func NewWorkerMetrics() WorkerMetrics {
 		histograms:      make(map[samplers.MetricKey]*samplers.Histo),
 		sets:            make(map[samplers.MetricKey]*samplers.Set),
 		timers:          make(map[samplers.MetricKey]*samplers.Histo),
+		cardinality:     samplers.NewCardinalityCount(),
```

Comment:
For posterity maybe note that we can't use a `map[MetricKey]HLL` here because we need a per-name map not a per-name, per-unique-tag map.